katcp/resource.py

```diff
@@ -595,10 +595,15 @@ def parent_name(self):
 
     @property
     def name(self):
-        """Name of this KATCP resource"""
+        """Name of this KATCPSensor"""
         return self._name
 
     @property
+    def python_id(self):
```

Comment:
OK - so shall we go with 'normalised_name' then. Seems like there are 3 half-votes for that one.